res2df/wellconnstatus.py

```diff
@@ -96,7 +96,7 @@ def fill_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
     parser.add_argument(
         "DATAFILE",
         type=str,
-        help="Name of Eclipse DATA file. " + "UNSMRY file must lie alongside.",
+        help="Name of reservoir DATA file. " + "UNSMRY file must lie alongside.",
```

Comment:
Can remove `" + "` and use e.g. `simulator input .DATA file` instead of `reservoir DATA file`.